packages/ui-library/src/components/forms/radio/radio-group/index.ts

```diff
@@ -59,15 +51,13 @@ export class BlrRadioGroup extends LitElement {
 
     const classes = classMap({
       [`${this.size}`]: this.size,
-      disabled: this.disabled || false,
-      readonly: this.readonly || false,
       checked: this.checked || false,
       error: this.hasError || false,
     });
 
-    const calculateOptionId = (label: string) => {
-      return label.replace(/ /g, '_').toLowerCase();
-    };
+    // const calculateOptionId = (label: string) => {
```

Comment:
Is this still required?

Reply:
No, I can remove it. I'm just not too sure what to do with this PR as a whole because I believe that @seemann was also working on a similar topic?

Reply:
If you dont need this changes, then I would say that we should not merge it. If we do so the radio group will not really work anymore as I descripted in Slack.